components/time/src/zone/mod.rs

```diff
@@ -193,8 +193,9 @@ pub(crate) mod ule {
     /// An indication of a period when a time zone maps to a specific UTC offset.
     ///
     /// The `Daylight` variant is defined to be the period that has a higher
-    /// offset from UTC than the `Standard` variant. This
-    /// behavior may or may not be intuitive; for example:
+    /// offset from UTC than the `Standard` variant. Although these definitions
+    /// correspond to daylight saving time as observed in some parts of the world,
+    /// the actual behavior may or may not be intuitive; for example:
```

Comment:
Let's not get carried away here, 3 out of 490 zones have exceptions

```suggestion
    /// offset from UTC than the `Standard` variant. Although these definitions
    /// correspond to daylight saving time as observed in most of the world,
    /// some zones exhibit unintuitive behavior:
```